pipeline/terraform/modules/es_index/main.tf

```diff
@@ -0,0 +1,26 @@
+locals {
+  full_name = var.name
+
+  mappings_file = "${var.config_path}/mappings.${var.mappings_name}.json"
+  analysis_name = var.analysis_name != "" ? var.analysis_name : var.mappings_name
+
+  analysis_json = jsondecode(local.analysis_name != "empty" ? file("${var.config_path}/analysis.${local.analysis_name}.json") : "{}")
```

Comment:
This was a little confusing at first, because it wasn't clear to me where `analysis_name` was being set to `"empty"` – I can see it's coming via `mappings_name`, but I wonder if we could document this expectation in the variable descriptions? Just a thought.

Reply:
Good point.  I wanted to avoid the situation where authors would have to type in two identical values, given that that would be the most common case, so set it to default to match mappings (then failed to describe that defaultness adequately).